osm_fieldwork/CSVDump.py

```diff
@@ -32,7 +32,11 @@
 from osm_fieldwork.osmfile import OsmFile
 from osm_fieldwork.xlsforms import xlsforms_path
 
+#Instantiate logger
+log_level = os.getenv("LOG_LEVEL", default="INFO")
```

Comment:
This like doesn't do anything. It gets the log level from the environment, but doesn't set it anywhere. I don't think we need it